OpenRA.Mods.D2k/Traits/Carryall.cs

```diff
@@ -50,7 +50,8 @@ public Carryall(Actor self, CarryallInfo info)
 
 			IsBusy = false;
 			IsCarrying = false;
-			carryHeight = self.Trait<Helicopter>().Info.LandAltitude;
+			var helicopter = self.Info.TraitInfoOrDefault<HelicopterInfo>();
+			carryHeight = helicopter != null ? helicopter.LandAltitude : WDist.Zero;
```

Comment:
For the attention of people who know better: Should Carryall require the helicopter trait instead of making this change, or is this change OK?